scss/2-tools/_interactive-elements.scss

```diff
@@ -15,3 +15,40 @@
     @include cads-interactive-focus();
   }
 }
+
+@mixin cads-nav-link-focus() {
+  background-color: $cads-palette__white;
```

Comment:
can these be swapped out for a `$cads-language` variable to make theming a bit easier? Perhaps out of scope.

Reply:
`$cads-language__brand-primary-background`